test/test_unary_ufuncs.py

```diff
@@ -78,9 +78,7 @@
 # The randomly generated values can be constracted by the domain
```

Comment:
I know this isn't part of this PR, but while we're here would you please fix "constracted" by replacing it with "restricted."

Reply:
Other parts of this comment will have to be updated, too, since the signature no longer contains "include_large_values" or "include_extremal_values."

Reply:
fixed